### PR TITLE
Add "sapply function" guide to R guides

### PR DESCRIPTION
**Pull request by nCarol595**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
@nCarol595 I have not used `r` before, so I am not familiar with the syntax, but I tried to run the code examples on replt.it, but not sure how to do that to validate your examples work as intended.  I thought I should be able to copy/paste the example and run it to see a result, but it does not seem to work that way.  Is everything you are writing in the code block actual code or is some of it pseudo-code?

File: guide/english/r/functions/sapply-function/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: sapply() Function
+---
+## sapply() Function
+
+The sapply() function is part of a family of apply() functions that aide in manipulating data. It takes 2 or more parameters and it returns a vector. The first parameter could be a vector or a data frame. The second parameter is usually a function you want to apply to the first parameter. 
+
+You can use sapply to get the max of every column in a data frame as a vector.
+```r
```
Comment (erictleung) [inline guide/english/r/functions/sapply-function/index.md]:
```suggestion
```r
df <- data.frame(a = c(49, 12, 32, 21, 16), b = c(22, 39, 8, 54, 26))
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Thanks for taking the time to write this up! Good to see that R is being represented, but just a few concerns and suggestions to make this article better. Hope the feedback is helpful.

**Comment by nCarol595** [comment]
@RandellDawson Thanks for the feedback, I will make the suggested changes to the article. Also thanks to @erictleung for the suggestions as well.

**Comment by RandellDawson** [comment]
@nCarol595 Are you able to make these changes?

**Comment by RandellDawson** [comment]
@erictleung Since the OP has not made the changes and it has been well over 21 days since we have last heard from the OP, I suggest you either make the suggested changes on behalf of the user or close this PR as stale.

**Comment by erictleung** [comment]
@RandellDawson I can take a look at this in the next couple days. Thanks!

**Review by erictleung — APPROVED** [review]
Changes made and code is minimally reproducible.

**Comment by erictleung** [comment]
@RandellDawson I've made the changes I've suggested. I think this should be a good start for this article. Let me know if you have any other suggestions on this starting article. Thanks!

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you @nCarol595 and @erictleung for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**